bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -2,18 +2,27 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% extends "mozorg/base-resp.html" %}
+{% extends "base-pebbles.html" %}
 
-{% add_lang_files "mozorg/plugincheck-redesign" %}
+{% add_lang_files "mozorg/plugincheck-redesign" "firefox/shared" %}
```

Comment:
For the newsletter signup box. We could also just add those strings as new ones here, but they're already translated in the shared file.